### ISSUE TITLE
Suggestion: Blazeborns interact with lava as others with water

### ISSUE THREAD

**Issue by maffe03** [apace100/origins-fabric#104]
To fit better with the "naturally immune to the perils of the nether" part, maybe blazeborns interact with lava, like how normal players interact with water. For example: not taking fall damage when falling from great heights into it, or having better vision in it. This will make traversing the nether a bit less annoying while still being a challenge.

**Comment by coolmaus** [comment]
I agree with you, I tried to install torrid vision mod to improve vision under lava but it seems incompatible.
